wrapper/xgboost.py

```diff
@@ -478,10 +490,16 @@ def load_model(self, fname):
 
         Parameters
         ----------
-        fname : string
-            Input file name.
+        fname : string of file handle
+            Input file name or file handle object.
         """
-        xglib.XGBoosterLoadModel(self.handle, c_str(fname))
+        if isinstance(fname, string_types):  # assume file name
+            xglib.XGBoosterLoadModel(self.handle, c_str(fname))
+        else:
+            buf = fname.getbuffer()
+            length = ctypes.c_ulong(buf.nbytes)
+            ptr = ctypes.byref(ctypes.c_void_p.from_buffer(buf))
```

Comment:
In my understanding, byref is a taking address operator in C, which means we are taking address of the object. When it is a c_long, it takes address of that lone. When it is c_void_p, it should have take the address of the that void_p